src/tests/ftest/server/metadata.py

```diff
@@ -280,6 +280,10 @@ def test_metadata_fillup(self):
             "Phase 3: passed (created %d / %d containers)", len(self.container), loop)
         self.log.info("Test passed")
 
+        # instruct tearDown to skip containers destroy
+        # (we want to invoke pool destroy with a full metadata rdb)
+        self.skip_destroy_containers = True
```

Comment:
There already exists a few mechanisms within the `TestContainer` class to disable calling either the pydaos or daos command container destroy commands, without the need to add yet another attribute to the `TestWithServers` class:
```suggestion
        # Skip containers destroy (we want to invoke pool destroy with a full metadata rdb)
        for container in self.container:
            container.container = None
```
or
```suggestion
        # Skip containers destroy (we want to invoke pool destroy with a full metadata rdb)
        for container in self.container:
            container.opened = False
            if container.container:
                container.container.attached = False
```

Alternatively, and possibly better as it would allow us to temporarily disable destroying containers with the easy option of re-enabling destroying containers, we could add a `TestContainer.skip_destroy` attribute:
```suggestion
        # Skip containers destroy (we want to invoke pool destroy with a full metadata rdb)
        for container in self.container:
            container.skip_destroy = True
```

Reply:
In addressing the feedback I started this way (TestContainer.skip_destroy) but then felt compelled in TestWithServers:destroy_containers() to follow the "Destroying containers" output with how many containers were successfully destroyed, failed to destroy due to some error, or were skipped due to this control - etc. It seemed the logic wasn't so clean and not very worthwhile. So I took an approach similar to Makito's suggested change, making the pool destroy (with full metadata rdb) an explicit final phase of the test that would output similarly to the previously executed phases 1-3 - and it avoids the destroy_containers portion of the teardown.